xarray/backends/api.py

```diff
@@ -827,6 +828,10 @@ def open_mfdataset(
         - 'override': if indexes are of same size, rewrite indexes to be
           those of the first object with that dimension. Indexes for the same
           dimension must have the same size in all objects.
+    attrs_file : str, optional
```

Comment:
since `paths` may be a sequence of `pathlib.Path` objects, we should allow passing one here, too:
```suggestion
    attrs_file : os.PathLike, optional
```
or
```suggestion
    attrs_file : str or pathlib.Path, optional
```